src/views/canvas/CanvasSeqBlock.js

```diff
@@ -265,7 +265,7 @@ const View = boneView.extend({
   },
 
   _getResidueAtMouseEvent: function(e) {
-    const coords = mouse.rel(e);
+    const coords = mouse.rel(e.originalEvent);
```

Comment:
Unrelated, but grabbed along the way. 
Same issue as https://github.com/pradeepnschrodinger/msa/pull/11/files#r1414503447.